overlord/configstate/transaction_test.go

```diff
@@ -122,6 +121,11 @@ func (s *transactionSuite) TestGetCachedWrites(c *C) {
 	err := s.transaction.Get("test-snap", "foo", &value)
 	c.Check(err, IsNil, Commentf("Expected 'test-snap' config to contain 'foo'"))
 	c.Check(value, Equals, "bar")
+
+	value = ""
+	err = s.transaction.GetMaybe("test-snap", "foo", &value)
+	c.Check(err, IsNil, Commentf("Expected 'test-snap' config to contain 'foo'"))
```

Comment:
If `err` is not nil here, it doesn't necessarily imply that the config is missing an entry for `foo`, right? Since it doesn't return an error at all in that case?